damus/Models/Wallet.swift

```diff
@@ -71,6 +72,9 @@ enum Wallet: String, CaseIterable, Identifiable {
         case .bitcoinbeach:
             return .init(index: 10, tag: "bitcoinbeach", displayName: "Bitcoin Beach", link: "bitcoinbeach://",
                          appStoreLink: "https://apps.apple.com/sv/app/bitcoin-beach-wallet/id1531383905", image: "bbw")
+        case .blixtwallet:
+            return .init(index: 11, tag: "blixtwallet", displayName: "Blixt Wallet", link: "blixtwallet:lightning:",
```

Comment:
I'm in the process of internationalizing all bundled user-facing strings in https://github.com/damus-io/damus/pull/204/files#diff-268b247d088cf6d43dd938bba548f3a9df59a37825e1f0a4e74f3663c4f0d4b5R72

It would be great if you could change `"Blixt Wallet"` to `NSLocalizedString("Blixt Wallet", comment: "Dropdown option label for Lightning wallet, Blixt Wallet.")` so that we don't need to go fix it up later.

Thanks!

Reply:
all good I got it all merged together